client/lib/signup/step-actions/index.js

```diff
@@ -437,28 +439,44 @@ export function setOptionsOnSite( callback, { siteSlug, siteTitle, tagline } ) {
 	);
 }
 
-export function setStoreFeatures( callback, { siteSlug } ) {
+export async function setStoreFeatures( callback, { siteSlug }, stepProvidedItems, reduxStore ) {
 	if ( ! siteSlug ) {
 		defer( callback );
 		return;
 	}
 
-	//Hard-code the theme slug for now, we'll use design selection step in a future iteration.
+	try {
+		const patternPost = await wpcom.req.get( {
+			path: `/sites/dotcompatterns.wordpress.com/posts/4348?http_envelope=1`,
+			apiNamespace: 'rest/v1.1',
+		} );
+
+		const newPage = await wpcom.req.post( {
+			path: `/sites/${ siteSlug }/pages`,
+			apiNamespace: 'wp/v2',
+			body: {
+				content: patternPost.content,
+				title: 'Home',
```

Comment:
Should this be wrapped in a `translate` call for localization?

Reply:
No, because it ends up being post data in the database (as opposed to a string handled by Calypso) – but it does bring up the question of how this stuff gets translated.

Reply:
Hm, looks like patterns and such from `dotcompatterns` is getting translated somehow – I can see them when running through the other intents in a different language. Let's leave the concern as handled, until we see actual issues in the wild.

Reply:
I'm flipping on this now – having it translated here allows it to appear properly in the user's UI immediately (and the content translations we do are for our own properties, not for individual users' content). @andres-blanco or @nelsonec87 , can someone make this update before merging please?